physics/Interstitials/UFS_SCM_NEPTUNE/GFS_surface_composites_pre.F90

```diff
@@ -111,7 +110,6 @@ subroutine GFS_surface_composites_pre_run (im, lkm, frac_grid, iopt_lake, iopt_l
               if (cice(i) >= min_lakeice) then
                 icy(i)    = .true.
                 islmsk(i) = 2
-                tisfc(i)  = max(timin, min(tisfc(i), tgice))
```

Comment:
Could you confirm if the intent of tisfc for sea ice will also be applied to lake ice?

Reply:
Mike Barlage confirmed that it should be the lake ice model to determine the ice temperature, not here in the atmospheric model. 

Shan

Reply:
As the PoC of the CLM lake model, @tanyasmirnova Could you confirm that?

Reply:
@mzhangw The CLM model sends ice temperature into tisfc(i) variable. However, the line you point to should not be used with the CLM model. The testing of the CLM model has been done with frac_grid = .false., therefore the specific for the CLM lake model code is in the part of the code when frac_land=.false.:

186             else ! Not ocean and not land
187               **is_clm** = lkm>0 .and. iopt_lake==iopt_lake_clm .and. use_lake_model(i)>0
188               if (cice(i) >= min_lakeice) then
189                 icy(i) = .true.
190                 **if(.not.is_clm) then**
191                   tisfc(i) = max(timin, min(tisfc(i), tgice))
192                 endif
193                 islmsk(i) = 2
194               else
195                 cice(i)   = zero
196                 hice(i)   = zero
197                 islmsk(i) = 0
198                 icy(i)    = .false.
199               endif
200               islmsk_cice(i) = islmsk(i)
201               flag_cice(i)   = .false.
202               **if(is_clm) then**
203                 wet(i) = .true.
204                 if (icy(i)) then
205                   tsfco(i) = max(tisfc(i), tgice)
206                 endif
207               else if(cice(i) < one) then
208                 wet(i) = .true. ! some open lake
209                 if (icy(i)) then
210                   tsfco(i) = max(tisfc(i), tgice)
211                 endif
212               endif
213             endif

Reply:
@mzhangw @tanyasmirnova Following up on this discussion, so the changes in this PR are OK and don't need to be modified for CLM, correct?